dist/zone-nativescript.js

```diff
@@ -0,0 +1,842 @@
+var Zone$1 = (function (global) {
```

Comment:
Please Remove the changes to the `dist` folder. We update the dist folder on releases only.